src/templates/case-study.jsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import { Link, graphql } from 'gatsby';
+import { noop } from 'lodash';
+import cx from 'classnames';
+import get from 'lodash/get';
+import { renderRichText } from 'gatsby-source-contentful/rich-text';
+import { BLOCKS } from '@contentful/rich-text-types';
+
+import { Layout } from '../components/Layout';
+import { createBemBlockBuilder } from '../utils';
+import { SubscriptionBanner } from '../components/SubscriptionBanner/SubscriptionBanner';
+import { ArrowLeft } from '../svg/arrowLeft';
+import { Download } from '../svg/download';
+import downloadIcon from '../svg/download.svg';
+
+import './case-study.scss';
+
+const getBlocksWith = createBemBlockBuilder(['case-page']);
+
+const CaseStudyTemplate = props => {
+  const caseStuday = get(props, 'data.contentfulCaseStudy');
+  const { title, industry, challenges, highlights, benefitsResults } = caseStuday;
+
+  const options = {
+    renderNode: {
+      [BLOCKS.PARAGRAPH]: (node, children) => {
+        return <p className={getBlocksWith('__column--description')}>{children}</p>;
+      },
+      [BLOCKS.LIST_ITEM]: (node, children) => {
+        return children.map((item, index) => (
+          <li className={getBlocksWith('__column--list--item')} key={index}>
+            {item}
+          </li>
+        ));
+      },
+      [BLOCKS.EMBEDDED_ASSET]: node => {
+        const { url, description } = node.data.target;
+        return <img className={getBlocksWith('__column--image')} src={url} alt={description} />;
+      },
+      [BLOCKS.HEADING_4]: node => {
+        return <h4 className={getBlocksWith('__column--title')}>{node.content[0].value}</h4>;
+      },
+      [BLOCKS.HEADING_6]: node => {
+        return <h6 className={getBlocksWith('__column--intro')}>{node.content[0].value}</h6>;
+      },
+    },
+  };
+
+  return (
+    <Layout location={props.location}>
```

Comment:
Location is not needed
```suggestion
    <Layout>
```

Reply:
Fixed.